app/models/course.rb

```diff
@@ -90,6 +90,17 @@ def open_for_applications?
     site_statuses.open_for_applications.any?
   end
 
+  def applications_open_from
+    site_statuses
+      .open_for_applications
```

Comment:
Does this need to be tested? eg Something showing that it picks the earliest application open date?

Reply:
I didn't test that particular branch, because I don't think it's at a high risk of breaking… if more people feel strongly for it I can add some.